students/nam_vo/lesson09/mailroom_oo/donor_models.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+"""
+Class responsible for donor data encapsulation.
+"""
+class Donor(object):
+    def __init__(self, name, donations=[0]):
+        # Name must be a string
+        if isinstance(name, str):
+            self._name = name
```

Comment:
you should be able to assign to `self.name` and utilize the check you already have as part of the property